bionty/base/entities/_disease.py

```diff
@@ -28,31 +46,38 @@ class Disease(PublicOntology):
         {doc_entities}
     """
 
+    @overload
```

Comment:
As discussed, the overloads are great but then let's roll them out for all entities. A few more suffer from the weird mixture of Literal versions